package/run.sh

```diff
@@ -20,7 +20,28 @@ else
     FORCE_RESTART=false
 fi
 
-env "INSTALL_K3S_FORCE_RESTART=${FORCE_RESTART}" "INSTALL_K3S_SKIP_DOWNLOAD=true" "INSTALL_K3S_SKIP_SELINUX_RPM=true" "INSTALL_K3S_SELINUX_WARN=true" installer.sh $@
+
+# work around for https://github.com/k3s-io/k3s/issues/2306
+# depends on journald being available on first boot, so this may not work on all providers (e.g. Digital Ocean)
+DELAY=10
+while ! env "INSTALL_K3S_FORCE_RESTART=${FORCE_RESTART}" "INSTALL_K3S_SKIP_DOWNLOAD=true" "INSTALL_K3S_SKIP_SELINUX_RPM=true" "INSTALL_K3S_SELINUX_WARN=true" installer.sh $@
+do
+    # Only use the logs for the latest restart of k3s, otherwise errors encountered after an ETCD join error will not be reported properly.
+    journalctl -u k3s.service > K3S_LOGS
```

Comment:
A bit pedantic, but maybe a more descriptive name like the following:
```
    journalctl -u k3s.service > system-agent-installer-k3s-service.[log/txt]
```
or
```
    journalctl -u k3s.service > k3s-service.[log/txt]
```

Reply:
I've changed the temporary file name to `k3s-service.txt`